WalletWasabi.Fluent/Views/SearchBar/SearchBarDropdown.axaml

```diff
@@ -98,7 +96,7 @@
                      ToolTip.Tip="{Binding Name}"
                      VerticalAlignment="Center"  TextTrimming="CharacterEllipsis" MaxWidth="150" />
           <TextBlock Grid.Column="2" Opacity="0.5" VerticalAlignment="Center" ToolTip.Tip="{Binding Description}" Text="{Binding Description}"
-                     TextTrimming="CharacterEllipsis" />
+                     TextTrimming="CharacterEllipsis" MaxWidth="227" />
```

Comment:
You located where is the problem well, good job!

Let's adjust the solution. The problem with this hardcoded value is it only works in this context. If I were to change the title of an item to a longer one, we would have to adjust again this value...
`TextTrimming` should work automatically without setting a MaxWidth, if this doesn't happen then probably there is a problem with the parent control, which is the `Grid`.

If you change the `Grid` to a `DockPanel` like this:
```xml
<DockPanel Background="Transparent" Margin="10">
  <PathIcon DockPanel.Dock="Left" Width="16" Height="16" Opacity="0.8" VerticalAlignment="Center" Foreground="{DynamicResource AcrylicTrimForeground}"
            Data="{Binding Icon, Converter={x:Static conv:NavBarIconConverter.Instance}}" />
  <TextBlock Margin="10 0 10 0" Text="{Binding Name}"
             ToolTip.Tip="{Binding Name}"
             VerticalAlignment="Center"  TextTrimming="CharacterEllipsis" MaxWidth="150" />
  <TextBlock Opacity="0.5" VerticalAlignment="Center" ToolTip.Tip="{Binding Description}" Text="{Binding Description}"
             TextTrimming="CharacterEllipsis" />
</DockPanel>
```

You will get the same look but also It will work without any hardcoded length.
(Also Grid is a very memory consuming control, we should only use it when there is no other choice, so this change will have also a slight performance improvement)

Reply:
Updated the code. Thank you